x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="FreeType-based font drawing library for X"
+DESCRIPTION="Xft provides a client-side font API for X applications, \
+making the FreeType font rasterizer available to X clients. Fontconfig \
+is used for font specification resolution. Where available, the RENDER \
+extension handles glyph drawing; otherwise, the core X protocol is used."
+HOMEPAGE="https://www.x.org/"
+COPYRIGHT="2000-2003 Keith Packard"
+LICENSE="libXft"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXft-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f5a3c824761df351ca91827ac221090943ef28b248573486050de89f4bfcdc4c"
+SOURCE_DIR="libXft-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+libVersion="4.11.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libxft$secondaryArchSuffix = $portVersion
+	lib:libxft$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
+	lib:libxrender$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libxft${secondaryArchSuffix}_devel = $portVersion
+	devel:libxft$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libxft$secondaryArchSuffix == $portVersion base
+	devel:libx11$secondaryArchSuffix
+	devel:libxau$secondaryArchSuffix
+	devel:libxcb$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:kbproto$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libx11$secondaryArchSuffix
+	devel:libxrender$secondaryArchSuffix
+	devel:renderproto$secondaryArchSuffix
+	devel:util_macros$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -i
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool library file
+	rm $libDir/libXft.la
+
+	prepareInstalledDevelLib libXft
+	fixPkgconfig
+
+	if [ -n "$secondaryArchSuffix" ]; then
+		rm -rf $binDir $documentationDir
```

Comment:
```diff
 	if [ -n "$secondaryArchSuffix" ]; then
-		rm -rf $binDir $documentationDir
+		rm -rf "$documentationDir"
+		maybe_manDir=
+	else
+		maybe_manDir="$manDir"
 	fi
```
(We can only put `$manDir/man3` in the devel package if we still have `$documentationDir`.)

Reply:
Fixed.